hooks/useNft.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+import { Keypair, PublicKey } from '@solana/web3.js'
+import useWallet from './useWallet'
+import {
+  keypairIdentity,
+  Metaplex,
+  Nft,
+  NftWithToken,
+  Sft,
+  SftWithToken,
+} from '@metaplex-foundation/js'
+
+export const useNft = (mint: PublicKey | undefined) => {
```

Comment:
nit: this seems like it wants to live a happy life as a `react-query` query. you can check the `queries` folder for reference. I will not enforce this change but I like `react-query` and suggest this.

Reply:
I dont think I even use this file anywhere, going to kill it. It was goign to die with helius anyways.